lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -2322,6 +2323,13 @@ instance LiftHandler ErrStartTimeLaterThanEndTime where
         , "'."
         ]
 
+instance LiftHandler PastHorizonException where
+    handler _ = apiError err400 PastHorizon $ mconcat
+        [ "The end time is past the horizon (due to uncertainty about the next"
+        , " hard fork)."
+        , " Wait for the node to finish syncing to the hard fork."
+        ]
```

Comment:
Questionn Should the eras be listed? (Note that it's a list of Eras)

Reply:
Sounds fine like this already.